### PR TITLE
Update working-with-the-nuget-registry.md

### PR DESCRIPTION
**Pull request by stprior**
The version number in he point on updating PackageReference doesn't match the PackageReference version in the example referred to.

### Why:

Closes [issue link]

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by japui** [comment]
Update working-with-the-nuget-registry.md

File: content/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry.md [hunk 1]
```diff
@@ -199,7 +199,7 @@ Using packages from {% data variables.product.prodname_dotcom %} in your project
 
 {% data reusables.package_registry.authenticate-step %}
 
-2. To use a package, add `ItemGroup` and configure the `PackageReference` field in the *.csproj* project file, replacing the `OctokittenApp` package with your package dependency and `1.0.0` with the version you want to use:
+2. To use a package, add `ItemGroup` and configure the `PackageReference` field in the *.csproj* project file, replacing the `OctokittenApp` package with your package dependency and `12.0.2` with the version you want to use:
```
Comment (vgrl) [inline content/packages/working-with-a-github-packages-registry/working-with-the-nuget-registry.md:202]: Adding to your proposed change, let's mention the two attributes to be changed by their name and value, to try and avoid confusion about what needs to be changed in this step.

```suggestion
2. To use a package, add `ItemGroup` and configure the `PackageReference` field in the *.csproj* project file. Replace the `OctokittenApp` value in `Include="OctokittenApp"` with your package dependency, and replace the `12.0.2` value in `Version="12.0.2"` with the version you want to use:
```

**Review by vgrl — COMMENTED** [review]
🎉 Thanks, @stprior, ⚡ I'm going to add one more detail to your change to remove any remaining confusion about what parameters need to be replaced in this step, and then I'll go ahead and merge this PR.

**Review by vgrl — APPROVED** [review]
👍 Looks good in preview.

**Pull request merged**